src/openzaak/components/zaken/tests/test_notifications_send.py

```diff
@@ -898,32 +898,33 @@ def _reset_debug():
             os.environ["DEBUG"] = "no"
             self.addCleanup(_reset_debug)
 
-    def test_invalid_notification_config_create(self, mock_notif):
-        self._configure_notifications("bla")
-
-        url = get_operation_url("zaak_create")
-        zaaktype = ZaakTypeFactory.create(concept=False)
-        zaaktype_url = reverse(zaaktype)
-        data = {
-            "zaaktype": f"http://testserver{zaaktype_url}",
-            "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduiding.openbaar,
-            "bronorganisatie": "517439943",
-            "verantwoordelijkeOrganisatie": VERANTWOORDELIJKE_ORGANISATIE,
-            "registratiedatum": "2012-01-13",
-            "startdatum": "2012-01-13",
-            "toelichting": "Een stel dronken toeristen speelt versterkte "
-            "muziek af vanuit een gehuurde boot.",
-            "zaakgeometrie": {
-                "type": "Point",
-                "coordinates": [4.910649523925713, 52.37240093589432],
-            },
-        }
-
-        response = self.client.post(url, data, **ZAAK_WRITE_KWARGS)
-
-        self.assertEqual(response.status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
-        self.assertFalse(Zaak.objects.exists())
-        mock_notif.assert_not_called()
+    # TODO APIClient doesnt seem to complain about base_url `bla/`
```

Comment:
Should this be removed or fixed (create an issue for this behavior at `ape_pie` issue tracker) ?

Reply:
Tbh I don't know how much we care about this, the client will complain anyway once you perform the request, because `Session.request` will complain about it